internal/rbd/controllerserver.go

```diff
@@ -547,6 +547,16 @@ func (cs *ControllerServer) DeleteVolume(ctx context.Context, req *csi.DeleteVol
 	}
 	defer cs.VolumeLocks.Release(rbdVol.RequestName)
 
+	found, _, err := rbdStatus(ctx, rbdVol, cr)
+	if err != nil {
+		klog.Errorf(util.Log(ctx, "failed getting information for image (%s): (%s)"), rbdVol, err)
+		return nil, status.Error(codes.Internal, err.Error())
+	}
+	if found {
+		klog.Errorf(util.Log(ctx, "rbd %s is still being used"), rbdVol)
```

Comment:
do we need to log it something like `rbd image %s is still ...`?

Reply:
@Madhu-1

Reply:
Yes @humblec it helps for debugging. and also the code was already present, This PR is just moving the check outside